clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -3725,7 +3746,7 @@ void testMonitorablePlugins(GroupProtocol groupProtocol) {
             assertTrue(metrics.containsKey(expectedInterceptorMetric));
```

Comment:
This `testMonitorablePlugins` missed to use `GroupProtocol` in the consumer config, could you also update the test for different protocol